### PR TITLE
feat: Product listing

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Commits have been duplicated, kindly ignore 

image
image

image
image

image
image

### CODE REVIEW

File: src/components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  min-width: fit-content;
+  padding: 0.5ch;
+  position: absolute;
+  background-color: inherit;
+}
+.card-container .btn-float img {
+  margin: 0;
+}
+.card-vertical .btn-float {
+  top: 2%;
```
Comment (shashidwivedi) [inline src/components/Card/card.css:59]: For smaller units, try going with px values.

**Review by shashidwivedi — APPROVED** [review]
Looks good. Can be merged

File: src/pages/products/index.jsx [hunk 2]
```diff
@@ -1,5 +1,153 @@
...
+  const navigate = useNavigate();
+  const isInArray = (array, id) =>
+    array.filter((item) => item["_id"] === id).length ? true : false;
+  const likeButtonClickHandler = (id, product) => {
+    isInArray(wishlist, id)
+      ? removeFromWishlist(product)
+      : addToWishlist(product);
+  };
+  const cartClickHandler = (id, product) => {
+    console.log(cart, wishlist);
```
Comment (rahulyadav139) [inline src/pages/products/index.jsx]: remove console.log statement.

**Review by VLeads — APPROVED** [review]
Looks good. Nothing extra to suggest

**Review by rahulyadav139 — APPROVED** [review]
Appreciation: everything is looking great. codes are readable. Great work.

**Comment by HarshnaNHaswani** [comment]
Thanks for you time and suggestions @shashidwivedi  @VLeads  @rahulyadav139! I'll make the necessary changes

**Pull request merged**